src/main/kotlin/org/rust/ide/fixes/ConvertCfgNotMultipleArgsToMultipleNotsFix.kt

```diff
@@ -0,0 +1,55 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.fixes
+
+import com.intellij.codeInspection.LocalQuickFixAndIntentionActionOnPsiElement
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiFile
+import org.rust.lang.core.psi.RsMetaItem
+import org.rust.lang.core.psi.RsPsiFactory
+
+class ConvertMalformedCfgNotPatternToCfgAllPatternFix(element: RsMetaItem) : LocalQuickFixAndIntentionActionOnPsiElement(element) {
+    private val fixText = run {
+        val metaItemArgsList = element.metaItemArgs?.metaItemList ?: return@run ""
+        if (metaItemArgsList.size < 2) {
+            return@run ""
+        }
+
+        val negatedArguments = metaItemArgsList.joinToString(", ") { "not(${it.text})" }
+
+        "Convert to `all($negatedArguments)`"
+    }
+
+    override fun getFamilyName(): String = "Fix `not` cfg-pattern"
+    override fun getText(): String = fixText
+
+    override fun isAvailable(project: Project, file: PsiFile, startElement: PsiElement, endElement: PsiElement): Boolean {
+        val metaItem = startElement as? RsMetaItem ?: return false
+        return metaItem.metaItemArgsList.size > 1
+    }
+
+    override fun invoke(project: Project, file: PsiFile, editor: Editor?, startElement: PsiElement, endElement: PsiElement) {
+        if (startElement !is RsMetaItem) return
+        val metaItemList = startElement.metaItemArgs?.metaItemList ?: return
+
+        val factory = RsPsiFactory(project)
+        val newItem = factory.createMetaItem(buildString {
+            append("all(")
+
+            append(metaItemList.joinToString(", ") {
+                "not(${it.text})"
+            })
+
+            append(")")
+        })
```

Comment:
It seems like it's possible to extract this code and unify it with the code in `fixText`